### PR TITLE
Add visual comp nb and nn doxa.

### PR DESCRIPTION
**Pull request by angela24680403**
This PR adds visual comp nb:
Screenshot 2023-11-21 at 23 41 10

And link to doxa website for NN.

### CODE REVIEW

**Review by TheRootOf3 — CHANGES_REQUESTED** [review]
Thanks, looking good!
One question about the reinforcement learning document you added. We already have 2 of these. Do we keep it like this or remove? I'll approve once this is resolved! 👀

image

File: our-initiatives/tutorials/reinforcement-learning.md
Comment (TheRootOf3) [inline our-initiatives/tutorials/reinforcement-learning.md:1]: Do we need this file?

Comment (angela24680403) [reply our-initiatives/tutorials/reinforcement-learning.md:1]: oh no sorry lets remove!

**Review by TheRootOf3 — APPROVED** [review]
Thanks, LGTM! 🚀

**Pull request merged**